expenses-manager-frontend/src/main/frontend/src/app/right-sidebar/right-sidebar.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="horizontal-container"> 
+  <h2>{{ userName }}</h2>
+  <mat-select *ngIf="router.url == '/home'" [(value)]="selectedCurrencyIndex" (selectionChange)="onCurrencyChange(currencies[$event.value].code)">
+    <mat-option *ngFor="let currency of currencies; let i = index" [value]="i"> 
+      <fa-icon *ngIf="currency.code == 'USD'" [icon]="['fas', 'dollar-sign']"></fa-icon>
+      <fa-icon *ngIf="currency.code == 'EUR'" [icon]="['fas', 'euro-sign']"></fa-icon>
+      {{ currency.code }} 
+    </mat-option>
+  </mat-select>
+</div>
+<mat-divider class="outer-divider"></mat-divider>
+<mat-list *ngIf="router.url == '/home'">
```

Comment:
Aceasta lista ar trebui sa aiba in partea stanga un border, asa cum este in Figma.